Containerfile.bpfman-agent.openshift

```diff
@@ -31,13 +29,12 @@ ARG TARGETPLATFORM
 WORKDIR /
 COPY --from=bpfman-agent-build /usr/src/bpfman-operator/bpfman-agent .
 
-# Install crictl
-RUN ${DNF_CMD} -y install wget tar gzip ca-certificates
-ARG VERSION="v1.31.0"
-RUN wget https://github.com/kubernetes-sigs/cri-tools/releases/download/${VERSION}/crictl-${VERSION}-linux-${TARGETARCH}.tar.gz
-RUN tar zxvf crictl-${VERSION}-linux-${TARGETARCH}.tar.gz -C /usr/local/bin
-RUN rm -f crictl-${VERSION}-linux-${TARGETARCH}.tar.gz
-RUN ${DNF_CMD} -y clean all
+RUN ls -lR /etc/pki
+
+RUN subscription-manager repos --enable=rhocp-4_DOT_17-for-rhel-9-x86_64-rpms
+
+RUN ${DNF_CMD} -y install wget tar gzip ca-certificates cri-tools && \
```

Comment:
I installed wget, tar and gzip so I could unpack the crictl-*-tar.gz, so we wouldn't need them if we're doing an RPM install of `crictl`.  I think you added ca-certificates -- possibly so you could remove `--no-check-certificate`.  Do we still need it?

Reply:
Re: ca-certificates. No, it shouldn’t be needed once we're just installing cri-tools. In fact, I believe we can remove all of these ancillary packages. The PR’s initial goal was primarily to see how the addition of the subscription-manager repository would pan out.